add-on/chrome/modules/loop-content-process.js

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
+// This file is loaded as a process script, it will be loaded in the parent
+// process as well as all content processes.
+
+const { utils: Cu } = Components;
+Cu.import("resource://gre/modules/XPCOMUtils.jsm");
+Cu.import("resource://gre/modules/Services.jsm");
+Cu.import("chrome://loop/content/modules/AboutLoop.jsm");
+
+function AboutLoopChildListener() {
+}
+AboutLoopChildListener.prototype = {
+  onStartup: function onStartup() {
+
+    if (Services.appinfo.processType ==
+        Services.appinfo.PROCESS_TYPE_CONTENT) {
```

Comment:
Is the reason this doesn't apply to the chrome process, the fact the messages don't get sent to the parent process? If so, it'd be nice to add a comment here to that affect.